{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure.py:19: Exception"}, "start": 1612328040069, "stop": 1612328040069, "uuid": "155260fc-1f7f-4b4b-a476-f0c8cda8930f", "historyId": "985c3d972d231f87d0914b89f78754fd", "testCaseId": "cb41bd139e929d9eef92825c451cc27a", "fullName": "test_allure#test_broken", "labels": [{"name": "suite", "value": "test_allure"}, {"name": "host", "value": "carolineyuandeMacBook-Pro.local"}, {"name": "thread", "value": "3317-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure"}]}